set.mm

```diff
@@ -793913,6 +794091,458 @@ regarding the multiplication ( ~ rngqiprnglin ) via ~ rngqiprnglinlem1
+       by AV, 16-Mar-2025.) $)
+    rngqiprngfulem3 $p |- ( ph -> U e. B ) $=
+      ( crng wcel cgrp rnggrp syl rngqiprngfulem2 rngqiprng1elbas rngcl syl3anc
+      co grpsubcl grpcld eqeltrid ) AHJIJGUQZMUQZICUQBUGABCFVBIRUFAFUHUIZFUJUIZ
+      NFUKULZAVDJBUIZVABUIZVBBUIVEABDEFGIJKLNOPQRSTUAUBUCUDUMZAVCIBUIVFVGNABFGI
+      KLNOPQRSTUNZVHBFGIJRSUOUPBFMJVARUEURUPVIUSUT $.
+
+    $( Lemma 4 for ~ rngqiprngfu .  (Contributed by AV, 16-Mar-2025.) $)
+    rngqiprngfulem4 $p |- ( ph -> [ U ] .~ = [ E ] .~ ) $=
+      ( cec wceq co wcel oveq2i a1i crng cabl rngabl syl rngqiprngfulem2 rnggrp
+      rngqiprng1elbas rngcl syl3anc grpsubcl ablsubsub4 ablnncan oveq1d 3eqtr2d
+      cgrp cbs cfv csg csubg cnsg cress crg ringrng rng2idlnsg rngqiprngghmlem1
+      eqeltrrid nsgsubg mpdan 2idlbas eleqtrd ringidcl subgsub ringgrpd eqeltrd
+      eqid wb rngqiprngfulem3 qusecsub syl22anc mpbird ) AHEUHJEUHUIZJHMUJZKUKZ
+      AWOIJGUJZIMUJZKAWOJJWQMUJZICUJZMUJZJWSMUJZIMUJWRWOXAUIAHWTJMUGULUMABCFMJW
+      SIRUFUEAFUNUKZFUOUKZNFUPUQZABDEFGIJKLNOPQRSTUAUBUCUDURZAFVHUKZJBUKZWQBUKZ
+      WSBUKAXCXGNFUSUQXFAXCIBUKXHXINABFGIKLNOPQRSTUTZXFBFGIJRSVAVBZBFMJWQRUEVCV
+      BXJVDAXBWQIMABFMJWQRUEXEXFXKVEVFVGAWRLVIVJZKAWRWQILVKVJZUJZXLAKFVLVJUKZWQ
+      KUKIKUKWRXNUIAKFVMVJUKXOAFKNOAFKVNUJLUNPALVOUKZLUNUKQLVPUQVSVQKFVTUQZAWQX
+      LKAXHWQXLUKZXFAJBFGIKLNOPQRSTVRWAZAXLFKLOPXLWHZWBZWCAIXLKAXPIXLUKZQXLLIXT
+      TWDUQZYAWCKFLMXMWQIUEPXMWHZWEVBALVHUKXRYBXNXLUKALQWFXSYCXLLXMWQIXTYDVCVBW
+      GYAWCWGAXDXOHBUKXHWNWPWIXEXQABCDEFGHIJKLMNOPQRSTUAUBUCUDUEUFUGWJXFBEKFMHJ
+      RUEUAWKWLWM $.
+
+    $( Lemma 5 for ~ rngqiprngfu .  (Contributed by AV, 16-Mar-2025.) $)
+    rngqiprngfulem5 $p |- ( ph -> ( .1. .x. U ) = .1. ) $=
+      ( co c0g cfv wceq oveq2i crng wcel rngqiprng1elbas rnggrp rngqiprngfulem2
+      a1i syl rngcl syl3anc grpsubcl rngdi syl13anc rngsubdi rngass cmulr c2idl
+      cgrp ressmulr oveqd crg cbs eqid ringidcl ringlidm syl2anc2 oveq1d eqtr3d
+      eqtrd oveq2d grpsubid syl2anc 3eqtrd oveq12d grplidd ) AIHGUHZIJIJGUHZMUH
+      ZICUHZGUHZFUIUJZICUHZIWGWKUKAHWJIGUGULURAWKIWIGUHZIIGUHZCUHZWMAFUMUNZIBUN
+      ZWIBUNZWRWKWPUKNABFGIKLNOPQRSTUOZAFVIUNZJBUNZWHBUNZWSAWQXANFUPUSZABDEFGIJ
+      KLNOPQRSTUAUBUCUDUQZAWQWRXBXCNWTXEBFGIJRSUTVAZBFMJWHRUEVBVAWTBCFGIWIIRUFS
+      VCVDAWNWLWOICAWNWHIWHGUHZMUHWHWHMUHZWLABFGMIJWHRSUENWTXEXFVEAXGWHWHMAWOJG
+      UHZXGWHAWQWRWRXBXIXGUKNWTWTXEBFGIIJRSVFVDAWOIJGAWOIILVGUJZUHZIAGXJIIAKFVH
+      UJZUNGXJUKOKFLGXLPSVJUSVKALVLUNILVMUJZUNXKIUKQXMLIXMVNZTVOXMLXJIIXNXJVNTV
+      PVQVTZVRVSWAAXAXCXHWLUKXDXFBFMWHWLRWLVNZUEWBWCWDXOWEVTABCFIWLRUFXPXDWTWFW
+      D $.
+
+    ${
+      rngqipring.p $e |- P = ( Q Xs. J ) $.
+      $( The product of the quotient with a two-sided ideal and the two-sided
+         ideal, which both are rings, is a ring.  (Contributed by AV,
+         16-Mar-2025.) $)
+      rngqipring $p |- ( ph -> P e. Ring ) $=
+        ( xpsringd ) AMECUIUDRUJ $.
```

Comment:
Do we need to keep such a single-step theorem?

Reply:
You are right, and ~rngqipring is not used anyway, so I have removed it.